os/pm/pm.h

```diff
@@ -137,7 +155,6 @@ struct pm_domain_s {
 	uint8_t mcnt;
 
 	/* accum - The accumulated counts in this time interval
-	 * thrcnt - The number of below threshold counts seen.
 	 */
 
 	int16_t accum;
```

Comment:
Any use of accum variable after removing pm_activity ? 
becuase accum variable was only increased by pm_activity.